pkg/registry/apis/query/plugins.go

```diff
@@ -46,7 +48,7 @@ func (s *pluginsStorage) NamespaceScoped() bool {
 }
 
 func (s *pluginsStorage) GetSingularName() string {
-	return example.DummyResourceInfo.GetSingularName()
+	return s.resourceInfo.GetSingularName()
```

Comment:
This plugin storage in the query api server I've seen before, but not questioned. What's the deal with it and is the idea it should continue to live here?

Reply:
This is the list of what datasource apiservers exist -- it should be used to route query requests to the right endpoints.

It is currently hardcoded in our multi-tenant depolyment, but should eventually be filled up from apidiscovery.

The frontend would use it [here](https://github.com/grafana/grafana/pull/86493/files/dadc71e49e91fbebdfbdd6ee6c754d9abc2142eb#diff-e66e8da9f7917f6cdf1356835e841715a074f285034ed29f6a5c7d1852fa97f7R226) to know where to send specific datasource queries.